features/signals_railway_signals.yaml

```diff
@@ -1838,6 +1842,44 @@ features:
       - { tag: 'railway:signal:combined', value: 'DE-BOStrab:h' }
       - { tag: 'railway:signal:combined:form', value: 'light' }
 
+  - description: Karlsruhe VBK Vorsignale
+    country: DE
+    icon:
+      match: 'railway:signal:distant:states'
+      cases:
+        - { regex: '^(.*;)?DE-VBK:fv1(;.*)?$', value: 'de/vbk/fv1' }
+        - { regex: '^(.*;)?DE-VBK:fv2(;.*)?$', value: 'de/vbk/fv2' }
+        - { regex: '^(.*;)?DE-VBK:fv3(;.*)?$', value: 'de/vbk/fv3' }
+    tags:
+        - { tag: 'railway:signal:distant', value: 'DE-VBK:fv' }
+        - { tag: 'railway:signal:distant:form', value: 'light' }
+
+  - description: Karlsruhe VBK Vorsignale
+    country: DE
+    icon: { default: 'de/vbk/fv0' }
+    tags:
+        - { tag: 'railway:signal:distant', values: ['DE-VBK:fv', 'DE-VBK:fv0'] }
+        - { tag: 'railway:signal:distant:form', value: 'sign' }
+
+  - description: Karlsruhe AVG Vorsignale
+    country: DE
+    icon:
+      match: 'railway:signal:distant_crossing:states'
+      cases:
+        - { regex: '^(.*;)?DE-AVG:bü201(;.*)?$', value: 'de/avg/bue201' }
+        - { regex: '^(.*;)?DE-AVG:bü201v(;.*)?$', value: 'de/avg/bue201v' }
```

Comment:
`de/avg/bue201` and `de/avg/bue201v` look very different, is this correct?

Reply:
Yes that is correct. Bü201v is only one light indicating go if illuminated and stop if not. Bü201 contains two lights. One for stop and one for go. I haven't touched `de/avg/bue201v` because it seems to fit into the current design language of BOStrab bar signals in OpenRailwayMap with the light for the horizontal bar being dark.

Reply:
Great, thanks!